news/index.html

```diff
@@ -85,6 +85,32 @@
             <div id="content" class="floe-content">
                 <div class="flc-toc-tocContainer toc"> </div>
                 <h2> News </h2>
+                
+                <div class="floe-news-item">
+                    <h3>Self-Assessment Learning Toolkit</h3>
+                    <p class="floe-date">August 21, 2017</p>
+                    <p>
+                        We have been working on the design and implementation of an inclusive self-assessment toolkit that will engage learners themselves – supported by personalized learning analytics – as aspiring research scientists in the important subject of self-regulation and self-determination. The toolkit will engage a broad spectrum of learners in a self-directed, personalized learning process, by enabling learners to set flexible goals and customizable milestones towards achieving those goals based on their unique needs and preferences. 
+                    </p>
+                    <p>
+                        Recently, the design team has been considering alternate entry points into the toolkit, from highly specific to broad and exploratory. Part of this work has has been the exploration of new ways of engaging the learner through customizable “touch-notes”, which would afford the learner a quick and easy manual data collection process. This could be combined with inputs from automated data collection sources (e.g. from a weather app or a personal health app, etc.), resulting in a fully personalizable data analysis and self-assessment.
+                    </p>
+                    <p>
+                        The following images show examples of the designs we are currently developing (select to enlarge):
+                    </p>
+                    <figure>
+                        <figcaption>Figure 1. A learner can choose to be prompted for more detailed information when they select a touchnote in their personalized collection.</figcaption>
```

Comment:
The same issues related to the `<figcaption>` and the alt text should be fixed here as well.